src/views/partials/subscribe/mailchimp.hbs

```diff
@@ -1,68 +1,101 @@
+      <div class="mc-field-group">
+        <label for="mce-EMAIL">E-Mail-Adresse <span class="asterisk">*</span>
+        </label>
+        <input type="email" value="" name="EMAIL" class="required email" id="mce-EMAIL">
       </div>
-      <div class="form-row">
-        <div class="col">
-          <label for="mce-FNAME">{{{__ "common_name"}}} </label>
-          <input type="text" value="" name="FNAME" class="form-control" id="mce-FNAME">
+      <div class="mc-field-group">
+        <label for="mce-FNAME">Vorname </label>
+        <input type="text" value="" name="FNAME" class="" id="mce-FNAME">
+      </div>
+      <div class="mc-field-group">
+        <label for="mce-LNAME">Nachname </label>
+        <input type="text" value="" name="LNAME" class="" id="mce-LNAME">
+      </div>
+      <div id="mergeRow-gdpr" class="mergeRow gdpr-mergeRow content__gdprBlock mc-field-group">
+        <div class="content__gdpr">
+          {{{__ "mailchimp_info"}}}
+          <fieldset class="mc_fieldset gdprRequired mc-field-group" name="interestgroup_field">
+            <label class="checkbox subfield" for="gdpr_7373"><input type="checkbox" id="gdpr_7373" name="gdpr[7373]"
+                value="Y" class="av-checkbox "><span>E-Mail</span> </label>
+          </fieldset>
+          {{{__ "mailchimp_policy"}}}
         </div>
-        <div class="col">
-          <label for="mce-LNAME">{{{__ "common_surname"}}} </label>
-          <input type="text" value="" name="LNAME" class="form-control" id="mce-LNAME">
+        <div class="content__gdprLegal">
+          {{{__ "mailchimp_en"}}}
         </div>
       </div>
-      <div id="mce-responses" class="clearfix mb-4">
+      <div id="mce-responses" class="clear">
         <div class="response" id="mce-error-response" style="display:none"></div>
         <div class="response" id="mce-success-response" style="display:none"></div>
       </div>
-      <input type="submit" value="{{{__ "subscribe_button"}}}" name="subscribe" id="mc-embedded-subscribe"
-        class="btn btn-primary">
       <!-- real people should not fill this in and expect good things - do not remove this or risk form bot signups-->
       <div style="position: absolute; left: -5000px;" aria-hidden="true"><input type="text"
           name="b_63598878eb7adee616350bca7_9367cfd8d5" tabindex="-1" value=""></div>
-
+      <div class="clear"><input type="submit" value="Subscribe" name="subscribe" id="mc-embedded-subscribe"
+          class="btn btn-secondary"></div>
     </div>
   </form>
 </div>
-<script type="text/javascript" src="//s3.amazonaws.com/downloads.mailchimp.com/js/mc-validate.js"></script>
-<{{!}}script type="text/javascript">(function ($) {
-  window.fnames = ['EMAIL', 'FNAME', 'LNAME', 'ADDRESS', 'PHONE'];
-  window.ftypes = ['email', 'text', 'text', 'address', 'phone'];
-  /*
-  * Translated default messages for the $ validation plugin.
-  * Locale: DE
-  */
-  if ('{{{__ "locale"}}}' === 'de') {
```

Comment:
And probably adapt copy this check too, since here I am including translations only for deutsch version